src/evaluate.cpp

```diff
@@ -217,7 +217,7 @@ namespace {
   };
 
   // KingProtector[PieceType-2] contains a bonus according to distance from king
-  const Score KingProtector[] = { S(-3, -5), S(-4, -3), S(-3, 0), S(-1, 1), S(0, 0), S(0, 0), S(0, 0) };
+  const Score KingProtector[] = { S(-3, -5), S(-4, -3), S(-3, 0), S(0, 0), S(0, 0), S(-1, 1), S(0, 0) };
```

Comment:
As we no longer need the entry for the king as a placeholder, I think we should get rid of it as the value is not used. The same for the king attack weights and the mobility bonus.

Reply:
It never occurred to me that the values for King were added by you. I just assumed they were from official-stockfish. I will delete.